spyder/plugins/pylint/main_widget.py

```diff
@@ -365,12 +365,15 @@ def _start(self):
         processEnvironment = QProcessEnvironment()
         processEnvironment.insert("PYTHONIOENCODING", "utf8")
 
-        # Needed due to changes in Pylint 2.14.0
-        # See spyder-ide/spyder#18175
         if os.name == 'nt':
+            # Needed due to changes in Pylint 2.14.0
+            # See spyder-ide/spyder#18175
             home_dir = get_home_dir()
             user_profile = os.environ.get("USERPROFILE", home_dir)
             processEnvironment.insert("USERPROFILE", user_profile)
+            # Needed for Windows installations using standalone Python and pip.
+            # See spyder-ide/spyder#19385
+            processEnvironment.insert("APPDATA", os.environ.get("APPDATA"))
```

Comment:
I think we should do this only if Spyder is not running under Anaconda. That way we'll avoid making this process to use packages installed out of Anaconda or its environments.